modules/i3s/src/i3s-attribute-loader.js

```diff
@@ -55,7 +55,7 @@ export async function loadFeatureAttributes(tile, featureId, options = {}) {
     attributeLoadPromises.push(promise);
   }
   try {
-    attributes = await Promise.all(attributeLoadPromises);
+    attributes = await handlePromises(attributeLoadPromises);
```

Comment:
Maybe add a comment explaining why we need to support failed promises?

Reply:
Ok. Actually some of attributes can be encoded badly or some attributes can't to be loaded. We should get as many attributes as possible to show available attributes for user.